IronGram/src/main/java/com/theironyard/entities/Photo.java

```diff
@@ -21,6 +23,14 @@
     @Column(nullable = false)
     String filename;
 
+    @Column(nullable = false)
+    @ColumnDefault("false")
+    private boolean publicPhoto;
+
+    @Column(nullable = false)
+    @ColumnDefault("10")
```

Comment:
If you are going to make 10 the default then you should probably not do all if statements in the controller.   Also, this column desperately needs some comments to let the user know what the scale is.  Is it 10 seconds? milliseconds? minutes?